x-pack/plugins/security_solution/public/management/cypress/e2e/mocked_data/policy_response.cy.ts

```diff
@@ -15,7 +15,8 @@ import { navigateToFleetAgentDetails } from '../../screens/fleet';
 import { EndpointPolicyResponseGenerator } from '../../../../../common/endpoint/data_generators/endpoint_policy_response_generator';
 import { descriptions } from '../../../components/policy_response/policy_response_friendly_names';
 
-describe('Endpoint Policy Response', () => {
+// FLAKY: https://github.com/elastic/security-team/issues/6518
+describe.skip('Endpoint Policy Response', () => {
```

Comment:
thanks for linking the relevant ticket - apologies for the flaky test, we will get it fixed

Reply:
The issue linked really has no info. about the flakiness or reference a failure in CI. What is going on with these tests? are they failing CI?